chuffed/vars/int-var.cpp

```diff
@@ -30,6 +30,8 @@ IntVar::IntVar(int _min, int _max) :
   , preferred_val(PV_MIN)
   , activity(0)
   , in_queue(false)
+    , SBPSValueSelection(false)
```

Comment:
Please rename it and all its occurrences to "sbps_value_selection" to be consistent with existing naming convention.